src/external/spglib.jl

```diff
@@ -44,21 +49,30 @@ spglib_cell(lattice, atoms) = first(spglib_cell_atommapping_(lattice, atoms))
     end
 
     # Ask spglib for symmetry operations and for irreducible mesh
-    spg_symops = spglib.get_symmetry(spglib_cell(lattice, atoms),
-                                     symprec=tol_symmetry)
-
+    spg_positions, spg_numbers = spglib_atoms(atoms)
+
+    spg_n_ops = ccall((:spg_get_multiplicity, SPGLIB), Cint,
+        (Ptr{Cdouble}, Ptr{Cdouble}, Ptr{Cint}, Cint, Cdouble),
+        lattice, spg_positions, spg_numbers, Cint(length(spg_numbers)), tol_symmetry)
+    
+    spg_rotations    = Array{Cint}(undef, 3, 3, spg_n_ops)
+    spg_translations = Array{Cdouble}(undef, 3, spg_n_ops)
+    
+    ccall((:spg_get_symmetry, SPGLIB), Cint,
+         (Ptr{Cint}, Ptr{Cdouble}, Cint, Ptr{Cdouble}, Ptr{Cdouble}, Ptr{Cint}, Cint, Cdouble),
+         spg_rotations, spg_translations, spg_n_ops, lattice, spg_positions, spg_numbers,
+         Cint(length(spg_numbers)), tol_symmetry)
+  
     # If spglib does not find symmetries give an error
-    if spg_symops === nothing
-        err_message=spglib.get_error_message()
+    if spg_n_ops == 0
+        err_message = spglib_get_error_message()
         error("spglib failed to get the symmetries. Check your lattice, use a " *
               "uniform BZ mesh or disable symmetries. Spglib reported : " * err_message)
     end
 
-    Stildes = [St for St in eachslice(spg_symops["rotations"]; dims=1)]
-    τtildes = [rationalize.(τt, tol=tol_symmetry)
-               for τt in eachslice(spg_symops["translations"]; dims=1)]
-    @assert length(τtildes) == length(Stildes)
-
+    Stildes = [Mat3{Int}(spg_rotations[:, :, i])' for i in 1:spg_n_ops]
```

Comment:
Add a comment like "transpose for row-major -> column-major" so that it's clear this is just an unpacking thing and not eg that spglib returns S instead of Stilde